docs/proposals/20191030-machine-health-checking.md

```diff
@@ -172,6 +179,11 @@ Out of band:
 
 ## Alternatives
 
+Considered having the MHC do the deletion itself. This was discarded since it
+did not allow for more complex controllers, for example the Kubeadm
+Control Plane, to account for the reconciliations strategies
+they require.
```

Comment:
Since the current implementation was implemented, I think we should call that out specifically here. The current text seems a bit revisionist.

Reply:
Hah, yes it did seem strange but it also seemed strange to be altering an implemented proposal 🤷 . I'll try to update it to make it more clear.